### PR TITLE
chore: rename umirc.ts => .dumirc.ts

### PR DESCRIPTION
**Pull request by kiner-tang**
- [X] rename umirc.ts => .dumirc.ts
- [X] update .dumirc.ts config
- [X] update README.md

### CODE REVIEW

**Comment by Wxh16144** [comment]
- [ ] `docs/index.md` 可以参考一下这个 https://github.com/react-component/motion/blob/master/docs/index.md
- [ ] `.dumirc.ts` 参考 https://github.com/react-component/motion/blob/master/.dumirc.ts, 添加 `favicons` 和 `themeConfig .logo`

**Comment by kiner-tang** [comment]
> * [ ]  `docs/index.md` 可以参考一下这个 https://github.com/react-component/motion/blob/master/docs/index.md
> * [ ]  `.dumirc.ts` 参考 https://github.com/react-component/motion/blob/master/.dumirc.ts, 添加 `favicons` 和 `themeConfig .logo`

好的👌🏻

File: package.json [hunk 1]
```diff
@@ -69,5 +69,9 @@
   "peerDependencies": {
     "react": ">=16.9.0",
     "react-dom": ">=16.9.0"
+  },
+  "resolutions": {
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0"
```
Comment (MadCcc) [inline package.json:75]: 为啥要 resolution？

Comment (kiner-tang) [reply package.json:75]: image
因为我们没有锁版本，如果安装版本的类型与react版本不对应会报这个错误

Comment (MadCcc) [reply package.json:75]: devDependencies 里直接拉到 react 18？

**Review by Wxh16144 — APPROVED** [review]

**Pull request merged**